code401challenges/src/main/java/code401challenges/stacksandqueues/Queue.java

```diff
@@ -0,0 +1,54 @@
+package code401challenges.stacksandqueues;
+
+public class Queue<T> {
+    Node<T> front;
+    Node<T> back;
+    T value;

```

Comment:
A `Qeueu` doesn't have a `value`, it should only have a `front` and `back` that point to nodes (and the nodes have values)